modules/custom/openy_system/js/openy_system.js

```diff
@@ -0,0 +1,14 @@
+(function ($) {
+  "use strict";
+
+  /**
+   * Add confirmation message after uninstall package form submit.
+   */
+  Drupal.behaviors.openy_confirm_form = {
+    attach: function (context, settings) {
+      $("#packages-uninstall-confirm").on('click', function ()  {
+        return confirm(Drupal.t("Are you sure you want to uninstall packages?"))
```

Comment:
I bet here we need different message. With explanation user is going to get content removed from the DB
@YMCA-GTC @ymcagtc please confirm